corehq/apps/cloudcare/static/cloudcare/js/formplayer/sessions/views.js

```diff
@@ -70,6 +71,76 @@ hqDefine("cloudcare/js/formplayer/sessions/views", function () {
             }
             return _.template($(id).html() || "");
         },
+        initialize: function (options) {
+            this.model = new Backbone.Model({
+                page: options.pageNumber + 1 || 1,
+                limit: parseInt($.cookie("sessions-per-page-limit")) || 10,
+            });
+            this.model.on('change', function () {
+                this.options.listSessions(this.model.get('page') - 1, this.model.get("limit"));
+                this.render.bind(this);
+            }.bind(this));
+        },
+        events: {
+            'click @ui.paginators': 'onClickPageNumber',
+            'click @ui.paginationGoButton': 'paginationGoAction',
+            'change @ui.sessionsPerPageLimit': 'onPerPageLimitChange',
+            'keypress @ui.paginationGoTextBox': 'paginationGoKeyAction',
+            'keypress @ui.paginators': 'paginateKeyAction',
```

Comment:
nit: about the way these are grouped by the type of event. Just thinking if it should be grouped by element so that, `paginators` click and keypress are together for better debug/undersanding
```
     'click @ui.paginators': 'onClickPageNumber',
     'keypress @ui.paginators': 'paginateKeyAction',
```

Reply:
Thanks for pointing this out. ae29eb2806012c9258ca29ddb16d103d2d928308